python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -213,6 +227,13 @@ def sources(self) -> "RootAssetSelection":
         )
         return self.roots()
 
+    @public
+    def source_assets(self) -> "SourceAssetSelection":
```

Comment:
I think we should call this `upstream_source_assets`. `source_assets` implies that it's filtering down the current selection to only include the entries that are source assets, but this is actually returning entries that aren't in the current selection, if I'm understanding correctly.

Reply:
Ok, good point